windows/windows_subsystem_for_linux/wsl_convert_binary_to_txt.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get the binary file from guest os to local and convert it to text format.
+# Parameters:
+#   source_path: the source file path on remote host
+#   file_name: source file name
+# Return:
+#   new_file: the text format file on local host
+
+- include_tasks: ../utils/win_get_file.yml
+  vars:
+    win_get_file_src_path: "{{ source_path }}"
+    win_get_file_dst_path: "{{ current_test_log_folder }}"
+
+- name: Set the fact of file
+  ansible.builtin.set_fact:
+    src_file: "{{ fetch_file.dest }}"
+    wsl_new_file: "{{ current_test_log_folder }}/{{ file_name }}"
+
+- name: Covert the binary file to text file
+  ansible.builtin.shell: "tr -d '\\0' < {{ src_file }} > {{ wsl_new_file }} "
+
+- name: Display the new file content
```

Comment:
If this task still causing log plugin error, we can remove this task since it's not mandatory and can access it from test log folder when debugging.

Reply:
Will remove it before merging into main.